### ISSUE TITLE
NPM package name for @app-config

### ISSUE THREAD

**Issue by joelgallant** [lobodpav/node-app-config#3]
Just wondering if this package is still maintained?

If so don't worry about it all. I maintain a library under the `@app-config` namespace [here](https://app-config.dev/). We would certainly like the ergonomics of `npx app-config`, but I totally understand if you'd like to keep this package alive :slightly_smiling_face: .

**Comment by lobodpav** [comment]
No, it's not maintained anymore. I stopped NodeJs development years ago and kept the lib alive just in case others would still find it useful.

When having a look at npmjs.org I can see 652 weekly downloads. That seems like a good reason to keep the lib available, to be honest.

I'll mark it as deprecated though.
